### PR TITLE
Relative links in README.md

### PR DESCRIPTION
**Pull request by Starefossen**
This PR updates relative links consistently in README.md so that links also works from the repository "front page" (https://github.com/kubernetes/minikube).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -226,16 +226,16 @@ $ minikube start --docker-env HTTP_PROXY=http://$YOURPROXY:PORT \
 
 Minikube uses [libmachine](https://github.com/docker/machine/tree/master/libmachine) for provisioning VMs, and [localkube](https://github.com/kubernetes/minikube/tree/master/pkg/localkube) (originally written and donated to this project by [RedSpread](https://redspread.com/)) for running the cluster.
 
-For more information about minikube, see the [proposal](https://github.com/kubernetes/kubernetes/blob/master/docs/proposals/local-cluster-ux.md).
+For more information about minikube, see the [proposal](./docs/proposals/local-cluster-ux.md).
```
Comment (Starefossen) [inline README.md]: Ops, this one was incorrect. Sorry.

**Comment by Starefossen** [comment]
I signed it!

**Comment by jimmidyson** [comment]
I just checked the updated links in your PR branch (https://github.com/Starefossen/minikube/tree/patch-1) & the links still don't work.

**Comment by Starefossen** [comment]
@jimmidyson yeah... that was unfortunate timing. Looks like relative links on GitHub recently stopped working as of github/markup#926.

**Comment by jimmidyson** [comment]
@Starefossen Haha such coincidence... I was trying to figure out why it wasn't working as it looks good to me.

**Comment by jimmidyson** [comment]
Looks like they've fixed it. LGTM

**Comment by jimmidyson** [comment]
Thanks @Starefossen!

**Comment by Starefossen** [comment]
Great 😄 Thank _you_ for making Kubernetes 💖 – and all this other great stuff 🚀

**Pull request merged**